packages/bitcore-wallet-service/test/utils.js

```diff
@@ -72,6 +73,38 @@ describe('Utils', function() {
       should.exist(res);
       res.should.equal(false);
     });
+    it('should call logger when _tryImportSignature throws', function() {
+      var logSpy = sinon.spy(logger,"error");
+      var res = Utils.verifyMessage('hola', null, '02555a2d45e309c00cc8c5090b6ec533c6880ab2d3bc970b3943def989b3373f16');
+      should.exist(res);
+      should.equal(logSpy.called,true);
+      res.should.equal(false);
+      logSpy.restore();
+    });
+    it('should call logger when _tryImportPublicKey throws', function() {
+      var logSpy = sinon.spy(logger,"error");
+      var res = Utils.verifyMessage('hola', '3045022100d6186930e4cd9984e3168e15535e2297988555838ad10126d6c20d4ac0e74eb502201095a6319ea0a0de1f1e5fb50f7bf10b8069de10e0083e23dbbf8de9b8e02785', null);
+      should.exist(res);
+      should.equal(logSpy.called,true);
+      res.should.equal(false);
+      logSpy.restore();
+    });
+    it('should call logger when _tryVerifyMessage throws', function() {
+      var logSpy = sinon.spy(logger,"error");
+      var fn = () => [];
+      var tryImportSigStub = sinon.stub(Utils, '_tryImportSignature').callsFake(fn);
+      var tryImportPubKeyStub = sinon.stub(Utils, '_tryImportPublicKey').callsFake(fn);
+
+      var res = Utils.verifyMessage('hola',[],[]);
+
+      should.exist(res);
+      should.equal(logSpy.called,true);
+      res.should.equal(false);
+
+      logSpy.restore();
+      tryImportSigStub.restore();
+      tryImportPubKeyStub.restore();
```

Comment:
We can actually clean these restores up a little by just doing `sinon.restore()`